src/components/templates/Audience/Audience.tsx

```diff
@@ -317,6 +319,8 @@ export const Audience: React.FC<AudienceProps> = ({ venue }) => {
       seated: userSeated,
     });
 
+    const shouldShowReactions = settingsAreLoaded && settings.showReactions;
```

Comment:
if settings are not loaded, this defaults `showReactions` to `false`, while so far it has been `true`. An explicit default should be used i.e.

```suggestion
    const shouldShowReactions = areSettingsLoaded ? settings.showReactions : DEFAULT_SHOW_REACTIONS;
```

Reply:
@goran-peoski-work, I believe the logic is correct here. We need to wait until the settings are loaded, and then decide if we want to show the reactions. The default value (`DEFAULT_SHOW_REACTIONS`) is already used in the `useSettings` hook, so if nothing is specified there, DEFAULT_SHOW_REACTIONS will play. 

It's done this way to avoid flickering. We'll better show reactions a second later but by then we'll be sure that we want them to be shown.

Reply:
> We need to wait until the settings are loaded,

Why the need?

> default value (DEFAULT_SHOW_REACTIONS) is already used in the useSettings hook, so if nothing is specified there, DEFAULT_SHOW_REACTIONS will play

@sunny-viktoryia  This means no check should be done if the settings are loaded since loaded or not, the default is provided and shouldn't be overriden by the `isLoaded` value.

Reply:
fixed in 2bf823d11ad27c3992680c9ac6a4585293744884